README.md

```diff
@@ -1,2 +1,50 @@
 # integration-tests
-Integration test suite for the Discovery release of the Enigma Protocol
+Integration test suite for the Discovery release of the Enigma Protocol.
+
+The contents of this repo are used in the [docker-environment](https://github.com/enigmampc/docker-environment) to create a `client` Docker image used to run these test automatically in our Continuous Integration (CI) environments.
+
+## Running tests locally
+
+This setup is only relevant for developers interested in manually debugging some of these tests, or wanting to tweak any particular test to adapt them to other applications: 
+
+1. Add an `.env` file with the following (and choose either `SW` or `HW` for `SGX_MODE)`
```

Comment:
Maybe add:
```bash
git@github.com:enigmampc/integration-tests.git integration-tests/integration-tests
cd integration-tests/integration-tests
```
So that `local_init.bash` won't pollute the parent directory of `integration-tests/`?

Reply:
👍